frontend/src/components/question-creation/question-elements/modals/short-answer/ShortAnswerModal.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import {
+  Button,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalCloseButton,
+  Text,
+  Divider,
+  ModalBody,
+  FormLabel,
+  FormControl,
+  Input,
+  FormErrorMessage,
+} from "@chakra-ui/react";
+
+interface ShortAnswerModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  onConfirm: (data: string) => void;
+  data?: string;
+}
+
+const ShortAnswerModal = ({
+  isOpen,
+  onClose,
+  onConfirm,
+  data,
+}: ShortAnswerModalProps): React.ReactElement => {
+  const [answer, setAnswer] = useState(data);
+  const [error, setError] = useState(false);
+
+  const handleConfirm = () => {
+    if (answer) {
+      onConfirm(answer);
+      setAnswer("");
+      setError(false);
+      onClose();
```

Comment:
```suggestion
      handleClose();
```
need to define `handleClose` before `handleConfirm`